tests/mocha/test_helpers/toolbox_definitions.js

```diff
@@ -4,7 +4,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-goog.module('Blockly.test.toolboxHelpers');
+goog.module('Blockly.test.helpers.toolbox_definitions');
```

Comment:
We generally seem to use camelcase for module names - maybe change to toolboxDefinitions?